tests/unit/hive_metastore/test_tables.py

```diff
@@ -308,3 +309,174 @@ def test_is_partitioned_flag():
         )
         in results
     )
+
+
+@pytest.mark.parametrize(
+    'table, mounts, describe, ddl, expected_hiveserde_type, expected_new_ddl',
+    [
+        # valid parquet hiveserde test
+        (
+            Table("hive_metastore", "schema", "table", "EXTERNAL", "HIVE", location="dbfs:/mnt/test_parquet/table1"),
+            [
+                Mount("/mnt/test_parquet", "s3://databricks/test_parquet"),
+                Mount("/mnt/test_orc", "s3://databricks/test_orc"),
+            ],
+            MockBackend.rows("col_name", "data_type", "comment")[
+                ("Serde Library", "org.apache.hadoop.hive.ql.io.parquet.serde.ParquetHiveSerDe", None),
+                ("InputFormat", "org.apache.hadoop.hive.ql.io.parquet.MapredParquetInputFormat", None),
+                ("OutputFormat", "org.apache.hadoop.hive.ql.io.parquet.MapredParquetOutputFormat", None),
+            ],
+            MockBackend.rows("createtab_stmt")[
```

Comment:
specify a constant `STATEMENT = MockBackend.rows("createtab_stmt")` and for other `MockBackend.rows` will reduce code duplication